static/sass/styles.scss

```diff
@@ -58,6 +58,7 @@ $color-social-icon-foreground: $color-light;
 @include vf-u-hide;
 @include vf-u-show;
 @include vf-u-layout;
+@include vf-u-vertical-spacing;
```

Comment:
I'd rather not include more vanilla CSS for this one case - I actually think is an issue with Vanilla. I think the base vanilla `dl` should probably have a default margin on it, like a `ul` - I'm not sure why it has the same styles as an `li`. We should file an issue on vanilla and add an override in snapcraft (with a comment linking to the created issue)